test/cli/other_test.py

```diff
@@ -1357,4 +1357,74 @@ def test_rule(tmpdir):
     lines = stderr.splitlines()
     assert lines == [
         "{}:4:0: style: found 'f' [rule]".format(test_file)
-    ]
\ No newline at end of file
+    ]
+
+
+def test_filelist(tmpdir):
+    list_dir = os.path.join(tmpdir, 'list-dir')
```

Comment:
I am curios why do you create a 'list-dir' folder instead of writing directly in the tempdir?

Reply:
No reason. Probably just so it is isolated.

Reply:
ok